core/src/test/scala/org/apache/spark/benchmark/BenchmarkBase.scala

```diff
@@ -25,7 +25,10 @@ import java.io.{File, FileOutputStream, OutputStream}
 abstract class BenchmarkBase {
   var output: Option[OutputStream] = None
 
-  def benchmark(): Unit
+  /**
+   * Main process of the whole benchmark.
+   */
+  def benchmarkSuite(): Unit
```

Comment:
I got the idea of this PR. But, for me, `benchmarkSuite` looks like a test suite again. Isn't it confusing in another way.

Reply:
Well it means a collection of benchmarks.  Just like `TestSuite` is a collection of test cases.

Reply:
For the word `suite`, it doesn't have to be about testing.